src/test/java/com/wikia/webdriver/testcases/globalshortcuts/KeyboardShortcutsTest.java

```diff
@@ -9,7 +9,7 @@
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePage;
 
-@Execute(asUser = User.USER, onWikia = "globalshortcuts-en")
```

Comment:
As long as ad behavior (that a closeable ad is covering a part of user functionality) is verified differently, LGTM